dapp/src/utils/evm/contracts.ts

```diff
@@ -0,0 +1,6 @@
+import { hololockerABI } from "../../generated";
+
+export const hololockerConfig = {
+  address: "0x963ba25745aEE135EdCFC2d992D5A939d42738B6",
```

Comment:
Should add a comment over this string pointing to https://github.com/PaimaStudios/PRC/blob/main/PRCS/prc-2.md#consistent-contract-address